src/processor/AliasProcessor.ts

```diff
@@ -8,6 +8,8 @@ import { InputStream, CommonTokenStream } from 'antlr4';
 import FSHLexer from 'fsh-sushi/dist/import/generated/FSHLexer';
 import FSHParser from 'fsh-sushi/dist/import/generated/FSHParser';
 
+const aliasRegex = /^\$?[a-zA-z0-9_\-\.]*$/;
```

Comment:
```suggestion
const aliasRegex = /^\$?[a-zA-z0-9_\-\.]+$/;
```
I think we want to require at least one character besides the `$`, right?

Reply:
Well... yes that seems reasonable. However, I took this regex from SUSHI. So I think that probably means SUSHI should also require the Alias has at least one other character. So I can update this one, but that probably means I should update SUSHI's as well.

Reply:
Updated in [b7e0415](https://github.com/FHIR/GoFSH/pull/254/commits/b7e04152c06aa03e829171edbc4f4a46d22708fb). I'll make a small PR in SUSHI (although I should probably run another regression, so that PR is coming eventually).